### PR TITLE
Added myself to the website

### PR DESCRIPTION
**Pull request by abachma2**
## Summary of changes

Added myself to the website in the grad student section

## Types of changes

- [ ] Bug fix (non-breaking change which fixes an issue)
- [X] New feature (non-breaking change which adds functionality)
- [ ] Breaking change (fix or feature that would cause existing functionality to change)
- [ ] I have read the **CONTRIBUTING** document.
- [ ] My code follows the code style of this project.
- [ ] My change requires a change to the documentation.
- [ ] I have updated the documentation accordingly.
- [ ] I have added tests to cover my changes.
- [ ] All new and existing tests passed.

## Associated Issues and PRs

- Issue: #197 

## Associated Developers

- Dev: @katyhuff 

## Checklist for Reviewers

Reviewers should use [this link](/manual/guides/pull_requests) to get to the 
Review Checklist before they begin their review.

### CODE REVIEW

**Comment by katyhuff** [comment]
I think you're a group member now, so you should have permission to add labels and projects to your PRs. It can also be helpful to look over the [PR checklist.](https://github.com/manual/guides/pull_requests)

**Comment by katyhuff** [comment]
You're so fast! Nice labels! I'll take a look at this PR. Looking good so far. Thanks for including labels/project.

File: _data/people.yml [hunk 1]
```diff
@@ -152,6 +152,21 @@
         icon: envelope-o
         name: email
 
+- name: Amanda Bachmann
+  pic: /img/people/BachmannA.png
```
Comment (katyhuff) [inline _data/people.yml:156]: For consistency with the other files*, can you please make the filename all lowercase? The file will also have to be changed with `git mv BachmannA.png bachmanna.png"

```suggestion
  pic: /img/people/bachmanna.png
```

* Also, in general, I am lazy and I prefer filenames to have no capital letters, when possible.

**Comment by abachma2** [comment]
I changed it in my local repo. I see that it has the commit in the chain for this PR, so it should all be changed as needed.

**Comment by katyhuff** [comment]
The line change in the yml file worked, but the filename change was not committed. This sometimes happens on case-insensitive platforms because they read `git mv BachmannA.png bachmanna.png` as `git mv same same` and do nothing. 

I can fix it on my machine.

**Pull request merged**